docs/concepts/move.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Move"
+linkTitle: "Move"
+weight: 1
+---
+
+## Moving Ondat volume instances
+
+A volume instance (replica or primary) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.
+
+It allows to manually re-balance a cluster, reduce load on over-loaded nodes, etc.
+
+## Example of use
+
+Here's an example of how the CLI cmd works:
+
+```bash
+storageos update volume move my-volume-id source-node-id destination-node-id --namespace my-namespace-name
+```
+
+## Moving to where an instance already exists
+
+When trying to move an Ondat volume instance into a node with another instance, the only scenario where this is valid is when the source node coincides with the primary and the destination node with the replica, in which case we take an approach of moving the "roles" as opposed to the data itself thus attempt to promote the replica on destination node to be the new primary instance of the Ondat volume.
```

Comment:
This sentence is quite long and a bit hard to understand. I'd rewrite it as:

If the node you are attempting to move a instance/deployment to already houses a replica we'll simply attempt to promote the replica deployment to a primary.

Reply:
Rewrote as suggested. I try to avoid the use of first person in between other details so it wasn't a 1-1 copy